Raining.rb

```diff
@@ -21,30 +21,34 @@
 def raining_weather
   puts "Is it raining today?"
   raining = gets.chomp
-if raining == "yes"
-  puts "raining"
-else 
-  puts "not raining"
+
+  if raining == "yes"
+    puts "raining"
+  else 
+    puts "not raining"
   end
+
   return raining
 end
 
 def temperature_weather
   puts "What's the temperature today?"
   temperature = gets.chomp
   temperature = temperature.to_i
+  is_raining = raining_weather()
 
-if temperature < 15 && raining_weather == "yes"
-       puts "It's wet and cold"
-elsif temperature < 15 && raining_weather == "no"
-       puts "It's not raining but cold"
-elsif temperature >= 15 && raining_weather == "no"
-       puts "It's warm but not rainning"
-else
-    puts "It's warm and raining"
+  if temperature < 15 && is_raining == "yes"
+         puts "It's wet and cold"
+  elsif temperature < 15 && is_raining == "no"
+         puts "It's not raining but cold"
+  elsif temperature >= 15 && is_raining == "no"
+         puts "It's warm but not rainning"
+  else
+      puts "It's warm and raining"
```

Comment:
@AnaBondiguel 
You can also rewrite this code this way:

```suggestion
  if is_raining == "yes"
    if temperature < 15
      puts "It's wet and cold"
    else
      puts "It's warm and raining"
    end
  else
    if temperature < 15
      puts "It's not raining but cold"
    else
      puts "It's warm and not raining"
  end
```

Reply:
![Screen Shot 2022-04-07 at 10 14 56 PM](https://user-images.githubusercontent.com/97415682/162196880-71106a93-7006-489d-a888-624e798cf399.png)
Thanks a lot for helping me with the code. When I run the code, I found the question "Is it raining today?" runs twice. How can get rid of it?